socs/smurf/MultiSlotController.py

```diff
@@ -0,0 +1,118 @@
+from ocs.ocs_client import OCSClient
+
+
+class MultiSlotController:
+    """
+    Simple client controller useful for managing all pysmurf-controller
+    agents that exist on a network. This can dispatch operations and check
+    operation status for any or all pysmurf-controllers on a network.
+
+    Args
+    -----
+    client_args: list
+        List of command line arguments to pass onto the OCSClient objects.
+
+    Attributes
+    ---------------
+    smurfs : dict
+        Dictionary containing OCSClient objects for each pysmurf-controller
+        on the network. This assumes pysmurf controller agents have
+        instance-id's of the form ``pysmurf-controller-<id>`` where id
+        can be the uxm-id or something like ``crate1slot2``. The keys of this
+        dict will be ``<id>``. This is populated from the registry agent on
+        init or using the funciton ``get_active_smurfs``.
+    """
```

Comment:
Since this likely will see wider usage at test institutions, can you add an Examples section to the end here with some simple examples.